SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestClientNetOwnership/NetOwnershipCube.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "NetOwnershipCube.h"
+#include "Net/UnrealNetwork.h"
```

Comment:
double empty line at top, splitting project, GDK and engine imports
```suggestion

#include "NetOwnershipCube.h"

#include "Net/UnrealNetwork.h"
```